src/main/java/io/github/thebusybiscuit/slimefunluckyblocks/SlimefunLuckyBlocks.java

```diff
@@ -192,6 +196,81 @@ private void registerSurprises() {
 		// Pandora Box Surprises
 		registerSuprise(new ReapersSurprise());
 		registerSuprise(new IronGolemsSurprise());
+
+		// CustomItem Surprises
+		if (cfg.getValue("custom") != null && cfg.getKeys("custom").size() > 0) {
+			for (String name : cfg.getKeys("custom")) {
+				LuckLevel luckLevel = LuckLevel.NEUTRAL;
+				List<ItemStack> items = new ArrayList<>();
+
+				if (cfg.getString("custom." + name + ".lucklevel") != null) {
+					try {
+						luckLevel = LuckLevel.valueOf(cfg.getString("custom." + name + ".lucklevel").toUpperCase());
+					} catch (IllegalArgumentException ex) {
+						getLogger().log(Level.WARNING, "Couldn't load lucklevel of CustomItem Surprise '" + name + "', now using NEUTRAL (default)");
+						getLogger().log(Level.WARNING, "Valid lucklevel types: LUCKY, NEUTRAL, UNLUCKY, PANDORA");
+					}
+				}
+
+				if (cfg.getValue("custom." + name + ".items") != null && cfg.getKeys("custom." + name + ".items").size() > 0) {
+					for (String itemID : cfg.getKeys("custom." + name + ".items")) {
+						ItemStack item = null;
+						String itemPath = "custom." + name + ".items." + itemID;
+						if (cfg.getString(itemPath + ".type") != null && Material.getMaterial(cfg.getString(itemPath + ".type")) != null) {
+							item = new ItemStack(Material.getMaterial(cfg.getString(itemPath + ".type")));
+							ItemMeta itemMeta = item.getItemMeta();
+
+							if (cfg.getInt(itemPath + ".amount") > 1) {
+								item.setAmount(cfg.getInt(itemPath + ".amount"));
+							}
+
+							if (cfg.getString(itemPath + ".displayname") != null) {
+								itemMeta.setDisplayName(ChatColor.translateAlternateColorCodes('&', cfg.getString(itemPath + ".displayname")));
+							}
+
+							if (cfg.getStringList(itemPath + ".lore").size() > 0) {
+								List<String> lore = new ArrayList<>();
+								cfg.getStringList(itemPath + ".lore").forEach(l -> {
+									lore.add(ChatColor.translateAlternateColorCodes('&', l));
+								});
+								itemMeta.setLore(lore);
+							}
+
+
+							if (cfg.getStringList(itemPath + ".enchants").size() > 0) {
+								cfg.getStringList(itemPath + ".enchants").forEach(ench -> {
+									String enchName = ench.split(":")[0];
+									String enchLevel = ench.split(":")[1];
+									Enchantment enchantment;
+									int level = 0;
+
+									if (Enchantment.getByName(enchName.toUpperCase()) == null) {
+										getLogger().log(Level.WARNING, "Couldn't set '" + enchName + "' enchant for CustomItem Surprise '" + name + "'");
+										return;
+									}
+									enchantment = Enchantment.getByName(enchName.toUpperCase());
+
+									try {
+										level = Integer.parseInt(enchLevel);
+									} catch (NumberFormatException ex) {
+										getLogger().log(Level.WARNING, "Couldn't set '" + enchName + "' enchant with level '" + enchLevel + "' for CustomItem Surprise '" + name + "'");
+										return;
+									}
```

Comment:
Relying on an Exception is rather bad practice, use a RegEx instead. You can use Slimefun's PatternUtils for this.